src/librustdoc/html/static/themes/ayu.css

```diff
@@ -61,6 +61,10 @@ pre {
 	background-color: #14191f;
 }
 
+.logo-container > img {
+	filter: drop-shadow(0 0 5px #fff);
```

Comment:
Do we have license agreement from @UtherII ? (Are you the same person here: https://www.reddit.com/r/rust/comments/hrfi8k/ayu_theme_is_now_available_on_nightly_rustdoc/fy4dzum/).
The POC doesn't have any license files: https://github.com/UtherII/poc_rustdoc .

Reply:
Yes I am. Do you want me to add a comment? It seems a bit too much for just a line of CSS but I can add it...

Reply:
I am not a lawyer too. I'm just afraid legal issues when UtherII doesn't grant us the permissions to copy the code.

Reply:
That seems to be a bit of long stretch considering the code was provided in an open discussion but if someone knows about it...

Reply:
No problem. You can use it

Reply:
Thanks a lot! Still very interested in what you'll come up with, with your own POC. :)